refactor/src/Cage.py

```diff
@@ -679,26 +618,12 @@ def update_distrib_discrete_subtract(distrib_delta, distrib):
             if distrib[geno] < 0:
                 distrib[geno] = 0
 
-    @staticmethod
-    def multiply_distrib(distrib: GenoDistrib, population: np.int64):
-        keys = distrib.keys()
-        values = list(distrib.values())
-        np_values = np.array(values)
-        if np.sum(np_values) == 0 or population == 0:
-            return dict(zip(keys, map(int, np.zeros_like(np_values))))
-        np_values = np_values * population / np.sum(np_values)
-        np_values = np_values.round().astype(np.int32)
-        # correct casting errors
-        np_values[-1] += population - np.sum(np_values)
-
-        return dict(zip(keys, map(int, np_values)))
-
-    def select_dams(self, distrib_dams_available: GenoDistrib, num_dams: int):
+    def select_dams(self, distrib_dams_available, num_dams):
```

Comment:
Is there a reason why this type annotation got removed?

Reply:
Most likely a merge omission - will add it back